configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
this just works with manager nodes, like if you put traefik on manager nodes this will be ok, but It's better to use TCP binding because you have sure that works in all types of nodes.